README.md

```diff
@@ -65,9 +65,10 @@ Note that the spawn generation is performed in the background on a timer, allowi
     * `rspawn.max_pregen` - maximum number of spawn points to pre-generate, default `20`
     * `rspawn.search_radius` - lateral radius around random point, within which a spawn point will be sought, default `32`
     * `rspawn.gen_frequency` - how frequently (in seconds) to generate a new spawn point, default `30`, increase this on slower servers
-* `rspawn.spawn_anywhere` - whether to spawn anywhere in the world at sea level
-    * default `true`
-    * if `false`, will randomize around the static spawn point
+* `rspawn.spawn_anywhere` - whether to spawn anywhere in the world at sea level (limited by the bounds spawn limits, check below)
```

Comment:
upss .. miscpelled here but ignore this. i'll fix later.. after you reviewed the hole pull merge request